src/main/kotlin/com/github/pedrovgs/maxibons/KarumiHQs.kt

```diff
@@ -1,10 +1,16 @@
 package com.github.pedrovgs.maxibons
 
 import arrow.Kind
-import arrow.data.extensions.list.foldable.foldM
+import arrow.core.extensions.list.foldable.foldM
+import arrow.typeclasses.ApplicativeError
 import arrow.typeclasses.Monad
+import kotlin.math.max
 
-class KarumiHQs<F>(private val M: Monad<F>, private val chat: Chat<F>) {
+class KarumiHQs<F>(
+    private val chat: Chat<F>,
+    private val M: Monad<F>,
+    AE: ApplicativeError<F, Throwable>
```

Comment:
You can merge these two together as just `monadError`

Reply:
Thanks @pakoito!